types/globalsat/LT-20/uplink.js

```diff
@@ -21,7 +21,7 @@ function consume(event) {
   var commandID = Bits.bitsToUnsigned(bits.substr(8, 8));
 
   data.longitude = (toLittleEndian(payload.substr(4, 6)) * 215) / 10 * 0.000001;
-  data.latitude = (toLittleEndian(payload.substr(10, 6)) * 215) / 10 * 0.000001;
+  data.latitude = (toLittleEndian(payload.substr(10, 6)) * 108) / 10 * 0.000001;
```

Comment:
are negative numbers covered with this change?

Reply:
no thats something i will look into ^^